cache/lru_cache.h

```diff
@@ -284,7 +284,11 @@ class ALIGN_AS(CACHE_LINE_SIZE) LRUCacheShard : public CacheShard {
   mutable port::Mutex mutex_;
 };
 
-class LRUCache : public ShardedCache {
+class LRUCache
+#ifdef NDEBUG
```

Comment:
Curious, why is this ifdef needed here. Whats the behavior otherwise?

Reply:
There is a unit test which overrides it.